pyproject.toml

```diff
@@ -79,7 +79,8 @@ dependencies = [
   "crash",
   "crate[sqlalchemy]>=0.34",
   "cratedb-toolkit==0.0.8",
-  "mlflow==2.10.2",
+  "joblib<1.4",
```

Comment:
## Problem
The patch https://github.com/crate-workbench/mlflow-cratedb/pull/124/commits/2d6db0d0de9898e86bb5ba41f63a8a3e6c409751 mitigates a problem with joblib 1.4.
```python
ImportError: cannot import name '_format_load_msg' from 'joblib.memory'
```

## References
- https://github.com/pycaret/pycaret/issues/3959